src/components/tx/SignOrExecuteForm/hooks.test.ts

```diff
@@ -566,24 +575,151 @@ describe('SignOrExecute hooks', () => {
       const { result } = renderHook(() => useAlreadySigned(tx))
       expect(result.current).toEqual(true)
     })
+
+    it('should return false if wallet has not signed a tx yet', () => {
```

Comment:
This change is unrealted. But I realised that the case was not nested in the matching `describe` block.